core-tests/src/test/java/org/mule/transformer/simple/RemoveSessionVariableTransformerTestCase.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.transformer.simple;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.internal.verification.VerificationModeFactory.times;
+import org.mule.api.MuleEvent;
+import org.mule.api.MuleSession;
+import org.mule.tck.size.SmallTest;
+
+import java.util.HashSet;
+
+@SmallTest
+public class RemoveSessionVariableTransformerTestCase extends AbstractRemoveVariablePropertyTransformerTestCase
+{
+    public RemoveSessionVariableTransformerTestCase()
+    {
+        super(new RemoveSessionVariableTransformer());
+    }
+
+    @Override
+    protected void addMockedPropeerties(MuleEvent mockEvent, HashSet properties)
```

Comment:
Why? these methods clearly need to receive a mock because of their name and what they do, so don't see an issue with using 'mockEvent'.